standalone-packages/sandpack/src/utils/generate-package-json.ts

```diff
@@ -25,13 +25,13 @@ export default function createMissingPackageJSON(
   if (!newFiles['/package.json']) {
     if (!dependencies) {
       throw new Error(
-        'No dependencies specified, please specify either a package.json or dependencies.'
+        'No dependencies specified, please specify either a package.json or provide dependencies.'
       );
     }
 
     if (!entry) {
       throw new Error(
-        "No entry specified, please specify either a package.json with 'main' field or dependencies."
+        "No entry specified, please specify either a package.json with 'main' field or provide entry."
```

Comment:
I would re-formulate this to: "Missing 'entry' parameter. Either specify an entry point, or pass in a package.json with the 'main' field set.

Reply:
Will update in sometime. Thanks for the review :)